### PR TITLE
[Beat] Face Payment Account Registration

### PR DESCRIPTION
**Pull request by sonadztux**
## Type of changes

 

Please check the type of change your PR introduces:
- [ ] Bugfix
- [x] Enhancement
- [ ] Code style update (formatting, renaming)
- [ ] Refactoring (no functional changes, no api changes)
- [x] Build related changes
- [x] Documentation content changes
- [ ] Other (please describe): 

## What is the current behaviour?
There is no backend API service for the process of creating a new face payment account with face liveness & face enrollment analytics.

## What is the new behavior?
Add backend API service for the process of creating a new face payment account with face liveness & face enrollment analytics.

## Checklist

- [ ] Lint and unit tests pass locally with my changes
- [ ] I have added tests that prove my fix is effective or that my feature works
- [x] I have added necessary documentation (if appropriate)
- [x] Any dependent changes have been merged and published in downstream modules
- [ ] I have filled the `[Unreleased]` [Changelog](https://github.com/nodefluxio/weber/blob/main/CHANGELOG.md

## Does this introduce a breaking change?
- [x] Yes
- [ ] No

- There is a new dependency module that is used, namely [gjson](https://github.com/tidwall/gjson) to make it easier to get values from cloud json responses. You need to run `go mod tidy` to update the module list on your local machine.
- The new endpoint `/api/v1/face-payment/account` handles two main mechanisms: to validate the new account data: check the phone already exists or not and create new face payment account (into db).

To validate the new account data
Validate new account data

To create new face payment account
Validate phone number

### CODE REVIEW

File: backend/controllers/face_payment.go [hunk 1]
```diff
@@ -0,0 +1,149 @@
...
+	}
+
+	// Insert new account data into db
+	var newAccount models.FacePaymentAccount
+	newAccount.SessionID = sessionId
+	newAccount.FullName = newAccountData.FullName
+	newAccount.Phone = newAccountData.Phone
+	newAccount.HaveTwin = newAccountData.HaveTwin
+	newAccount.CreatedAt = time.Now()
+	newAccount.UpdatedAt = newAccount.CreatedAt
```
Comment (lazuardyk) [inline backend/controllers/face_payment.go:115]: Better tambahin validasi lagi utk panjang karakternya (min, max) atau required juga kalo perlu.
Krn kalo dari frontend doang bisa diakalin

**Review by lazuardyk — APPROVED** [review]
Nice, tp menurut gua perlu ditambahin validasinya nanti,
Kalo ga kekejer merge dulu gapapa, gua approve

**Review by bagasteguhimani — APPROVED** [review]
Work!

File: backend/controllers/face_payment.go (hunk 1 above)
Comment (sonadztux) [reply backend/controllers/face_payment.go]: done a https://github.com/nodefluxio/weber/pull/79/commits/96ccd7ee9a02ca2b7ac3ece7009473cd44e82849 https://github.com/nodefluxio/weber/pull/79/commits/c7a37547aa8e630cd5e9d9926c16b6a7efbc8686

**Comment by sonadztux** [comment]
cc @williammfu check ada update request payload buat hit pertama ya, buat validasi inputan full name, phone, sama have twin.

**Pull request merged**